dls-controls/malcolmjs
# Feature/fast update graph perf #317

## Description

Fixed the rendering of the nav bar during fast attribute updates.
Fixed various items from the sprint 10 reivew

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- View the graph for a fast updating attribute

## Agile board tracking
